container-engine-lib/lib/backend_interface/objects/service_directory/persistent_directories.go

```diff
@@ -1,13 +1,19 @@
 package service_directory
 
 type DirectoryPersistentKey string
+type DirectoryPersistentSize int64
```

Comment:
you could use uint64 instead here

Reply:
create Volume in both Docker k8s works with signed integers :/